airflow/providers/google/cloud/hooks/dataproc.py

```diff
@@ -1192,6 +1194,12 @@ def wait_for_batch(
 
         return result
 
+    def check_error_for_resource_is_not_ready_msg(self, error_msg: str) -> bool:
+        """Check that reason of error is resource is not ready."""
+        key_words = ["The resource", "is not ready"]
```

Comment:
Is guaranteed "The resource" always started from the capital letter?

Reply:
@olegkachur-e in current moment it looks like it's guaranteed https://cloud.google.com/compute/docs/troubleshooting/troubleshooting-vm-creation#simultaneous_resource_mutation_or_creation_operations , but I don't know what will be in the future.